jobbergate-agent/jobbergate_agent/jobbergate/schemas.py

```diff
@@ -78,3 +78,45 @@ class SlurmJobData(pydantic.BaseModel, extra="ignore"):
     job_state: Optional[str] = None
     job_info: Optional[str] = None
     state_reason: Optional[str] = None
+
+
+class InfluxDBMeasurement(TypedDict):
+    """
+    Map each entry in the list returned by `InfluxDBClient(...).get_list_measurements(...)`.
+    """
+
+    name: INFLUXDB_MEASUREMENT
+
+
+class InfluxDBMeasure(TypedDict):
+    """
+    Map each entry in the generator returned by InfluxDBClient(...).query(...).get_points().
+    """
+
+    time: int
+    host: str
+    job: str
+    step: str
+    task: str
+    value: float
+    measurement: INFLUXDB_MEASUREMENT
+
+
+class JobSubmissionMetricsMaxTimes(pydantic.BaseModel):
+    """
+    Model for the max_times field of the JobSubmissionMetricsMaxResponse.
+    """
+
+    max_time: int
+    node_host: str
+    step: int
+    task: int
+
+
+class JobSubmissionMetricsMaxResponse(pydantic.BaseModel):
```

Comment:
yeah, I thought a bit difficult naming these because they are strictly related to API usage. The names only make sense when you read the docstring as well.

Reply:
I accept suggestions here on how to improve this

Reply:
I stared at it for a while and didn't come up with anything. So, I suppose it's fine.